modules/install-sync-gateway/install-sync-gateway

```diff
@@ -153,6 +155,9 @@ TimeoutSec=60
 WantedBy=multi-user.target
 EOF
 
+  # Remove the original log paths to reduce confusion
+  sudo rm -rf /var/log/sync_gateway
```

Comment:
Is this really a good idea? Should we `mv` instead?

Reply:
Oh this was already there and I just shifted this around. But I suspect you are right.